source/gui/settingsDialogs.py

```diff
@@ -2494,6 +2501,9 @@ def makeSettings(self, settingsSizer):
 		self.detectFormatAfterCursorCheckBox.SetValue(config.conf["documentFormatting"]["detectFormatAfterCursor"])
 		sHelper.addItem(self.detectFormatAfterCursorCheckBox)
 
+	def onLineIndentationChange(self, evt: wx.CommandEvent) -> None:

```

Comment:
This should probably remain private

```suggestion
	def _onLineIndentationChange(self, evt: wx.CommandEvent) -> None:
```